content/en/docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -173,6 +173,15 @@ Vagrant typically assigns two interfaces to all VMs. The first, for which all ho
 
 This may lead to problems with flannel. By default, flannel selects the first interface on a host. This leads to all hosts thinking they have the same public IP address. To prevent this issue, pass the `--iface eth1` flag to flannel so that the second interface is chosen.
 
+It could be fixed with the add of an extra row in this file: /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
```

Comment:
Completely agree. Changing file that comes from package or automatically generated is a bad suggestion. systemd supports drop-ins, thus if something needed to be adjusted, better to do it in custom drop-in.

Reply:
but that's the config of kubeadm, for me is the natural position of the canche that work on the structure of it, you had\can change it when you set the cgroup to put that like the docker cgroup

Reply:
it is a bug in the package that this file installed in /etc/systemd instead of /usr/lib/systemd. Better not to encourage users to modify it, but teach how to do systemd drop-ins with overrides.